config/locales/server.en.yml

```diff
@@ -1690,8 +1690,6 @@ en:
     invalidate_inactive_admin_email_after_days: "Admin accounts that have not visited the site in this number of days will need to re-validate their email address before logging in. Set to 0 to disable."
     include_secure_categories_in_tag_counts: "When enabled, count of topics for a tag will include topics that are in read restricted categories for all users. When disabled, normal users are only shown a count of topics for a tag where all the topics are in public categories."
     display_personal_messages_tag_counts: "When enabled, count of personal messages tagged with a given tag will be displayed."
-    top_menu: "Determine which items appear in the homepage navigation, and in what order. Example latest|new|unread|categories|top|read|posted|bookmarks"
```

Comment:
Sorry @lis2 this is my bad I worded this wrong, I meant only remove the "Example like|edit|flag|delete|share|bookmark|reply" part of these type of settings. There is no need for it now that you only are allowed to choose some choices

Reply:
I am not sure if there are other settings like this which have the "example" part or just top_menu and post_menu